arbor/mc_cell_group.cpp

```diff
@@ -322,18 +322,19 @@ void run_samples(
 void mc_cell_group::advance(epoch ep, time_type dt, const event_lane_subrange& event_lanes) {
     time_type tstart = lowered_->time();
 
+    // Bin and collate deliverable events from event lanes.
+
     PE(advance_eventsetup);
     staged_events_.clear();
 
-    fvm_index_type ev_begin = 0, ev_mid = 0, ev_end = 0;
-    // skip event binning if empty lanes are passed
+    // Skip event handling if nothing to deliver.
     if (event_lanes.size()) {
-
         std::vector<cell_size_type> idx_sorted_by_intdom(cell_to_intdom_.size());
         std::iota(idx_sorted_by_intdom.begin(), idx_sorted_by_intdom.end(), 0);
         util::sort_by(idx_sorted_by_intdom, [&](cell_size_type i) { return cell_to_intdom_[i]; });
 
         /// Event merging on integration domain could benefit from the use of the logic from `tree_merge_events`
+        fvm_index_type ev_begin = 0, ev_mid = 0, ev_end = 0;
```

Comment:
Are these variables used?

Reply:
Yes: see the intdom event merging code just below...

Reply:
No worries... I can't read a diff apparently.
They weren't in the added code, and I got confused looking for them.